packages/atomic/cypress/e2e/smart-snippet.cypress.ts

```diff
@@ -254,33 +251,34 @@ describe('Smart Snippet Test Suites', () => {
         .init();
     });
 
-    SmartSnippetAssertions.assertShowMore(true);
-    SmartSnippetAssertions.assertShowLess(false);
-    SmartSnippetAssertions.assertCollapseWrapperHeight(heightWhenCollapsed);
-    CommonAssertions.assertAccessibility(smartSnippetComponent);
+    it('should render correctly', () => {
+      SmartSnippetAssertions.assertShowMore(true);
+      SmartSnippetAssertions.assertShowLess(false);
+      SmartSnippetAssertions.assertCollapseWrapperHeight(heightWhenCollapsed);
+    });
 
-    describe('then pressing show more', () => {
-      beforeEach(() => {
-        SmartSnippetSelectors.showMoreButton().click();
-      });
+    CommonAssertions.assertAccessibility(smartSnippetComponent);
 
+    it('should work correctly when pressing show more and show less', () => {
+      SmartSnippetSelectors.showMoreButton().click();
+      SmartSnippetSelectors.collapseWrapperComponent().should(
+        'have.class',
+        'expanded'
+      );
       SmartSnippetAssertions.assertShowMore(false);
       SmartSnippetAssertions.assertShowLess(true);
       SmartSnippetAssertions.assertAnswerHeight(height);
-
-      describe('then pressing show less', () => {
-        beforeEach(() => {
-          SmartSnippetSelectors.showLessButton().click();
-        });
-
-        SmartSnippetAssertions.assertShowMore(true);
-        SmartSnippetAssertions.assertShowLess(false);
-        SmartSnippetAssertions.assertCollapseWrapperHeight(heightWhenCollapsed);
-        CommonAssertions.assertAccessibility(smartSnippetComponent);
-      });
+      SmartSnippetSelectors.showLessButton().click();
```

Comment:
Even though I see why we don't want to nest a bunch of `describe`s, but perhaps we should still make it clear as to where each "section" is with by adding some comments.